semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -546,5 +548,6 @@ def attributes(self):
         MetricSemanticConvention,
         UnitSemanticConvention,
         ScopeSemanticConvention,
+        AttributeGroupConvention
```

Comment:
```suggestion
        AttributeGroupConvention,
```
see https://github.com/open-telemetry/build-tools/actions/runs/3433329456/jobs/5723545460